core/src/main/java/org/apache/cxf/common/util/SortedArraySet.java

```diff
@@ -99,9 +103,9 @@ public boolean addAll(Collection<? extends T> c) {
         return val;
     }
     public boolean containsAll(Collection<?> c) {
```

Comment:
This method is unused in the CXF code base as far as I can tell.

Reply:
We probably could make it even better by returning as early as possible (upon finding first non-containing element)?

Reply:
Since class is public, it could be used outside of CXF codebase ...